.github/workflows/build-test.yaml

```diff
@@ -1448,9 +1448,94 @@ jobs:
           echo "::notice ::support bundle uploaded to aws replicated-dev account s3://kots-e2e-build-test-support-bundles/${BUNDLE}"
 
 
+  validate-helm-install-order:
+    runs-on: ubuntu-20.04
+    needs: [ can-run-ci, build-kots, build-kurl-proxy, build-migrations, push-minio, push-postgres ]
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s_version: [ v1.23.3-k3s1 ]
```

Comment:
since this test does not use testim, let's run it on all 4 supported k8s/k3s versions (like the other tests that do not use testim)

Reply:
I'm fine with running it on all of them, I just didn't think it needed to be there. I'll make the change